nilearn/_utils/niimg.py

```diff
@@ -110,7 +110,7 @@ def load_niimg(niimg, dtype=None):
     -----------
 
     niimg: Niimg-like object
-        See https://nilearn.github.io/stable/manipulating_images/input_output.html  # noqa:E501
+        See https://nilearn.github.io/stable/manipulating_images/input_output.html.  # noqa: E501
```

Comment:
I am not sure how it would effect rendering if there's a period at the end of URL

Reply:
I was trying to make things consistent, but yea it doesn't affect rendering.